Library/SharedFunctions.swift

```diff
@@ -257,6 +257,19 @@ public func discoveryPageBackgroundColor() -> UIColor {
   }
 }
 
+public func isNativeRiskMessagingControlEnabled() -> Bool {
+  guard let variant = AppEnvironment.current.optimizelyClient?
+    .variant(for: .nativeRiskMessaging)
+  else { return true }
+
+  switch variant {
+  case .control, .variant2:
```

Comment:
seems like we're only testing `control` and `variant1`, so can probably remove `variant2`?

Reply:
Ah so if I remove that we have to include a default which is also just returning true so I didn't want to add those two extra lines. We follow a similar convention for the other experiment.